pyproject.toml

```diff
@@ -77,6 +77,11 @@ pyproject-build = "build.__main__:entrypoint"
 [project.entry-points."pipx.run"]
 build = "build.__main__:entrypoint"
 
+[tool.flit.sdist]
+include = ["tests/", ".gitignore", "CHANGELOG.rst", "docs/", ".dockerignore", "tox.ini"]
+exclude = ["tests/__pycache__", "docs/_build"]
```

Comment:
Should be able to use `"**/__pycache__"` now.

Reply:
Okay, did require a bump to flit-core 3.8+.